book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -299,6 +299,16 @@ This changes the SHA-1s of the three most recent commits in your list, so make s
 Notice that the last commit (`f7f3f6d`) in the list is unchanged.
 Despite this commit being shown in the script, because it was marked as ``pick'' and was applied prior to any rebase changes, Git leaves the commit unmodified.
 
+[[_practical_exercises_for_git_rebase ]]
+==== Practical exercises for git rebase
```

Comment:
I don't think having an entire section just for a `NOTE` makes sense. Let's drop these few lines.

Reply:
Okay, I'll drop the header and chapter reference then!